src/elements/hx-toggle/index.js

```diff
@@ -0,0 +1,28 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
+
+import shadowMarkup from './_shadow.html';
+import shadowStyles from './_shadow.scss';
+
+/**
+ * Defines behavior for the `<hx-toggle>` element.
+ * NOTE: `<hx-toggle>` can have various default options, or it can be overriden.
+ *
+ * @extends HXFormControlElement
+ * @hideconstructor
+ * @since 0.24.0
+ */
+export class HXToggleElement extends HXFormControlElement {
```

Comment:
Please update:

```js
export class HXToggleElement extends HXElement {
```

Reply:
Changes have been done, and i have changed the import as well to support the above export . please review.